Maestro/subscriptions.json

```diff
@@ -640,7 +640,10 @@
         "https://github.com/dotnet/versions/blob/master/build-info/docker/debian/jessie.txt",
         "https://github.com/dotnet/versions/blob/master/build-info/docker/debian/stretch.txt"
       ],
-      "action": "dotnet-docker-nightly-pipebuild"
+      "action": "dotnet-docker-nightly-pipebuild",
+      "actionArguments": {
+        "vsoSourceBranch": "master"
```

Comment:
Yeah, I assumed the VSTS `Get Sources` default (which we rely on for other subscriptions) would be applied uniformly in the build, but I guess not to variables.